variables.tf

```diff
@@ -469,3 +469,47 @@ variable "graphdb_enable_userdata_scripts_on_reboot" {
   type        = bool
   default     = false
 }
+
+# KMS CMK
+
+variable "enable_cmk" {
+  description = "Enable CMK for encryption. If false, use AWS managed key."
+  type        = bool
+  default     = false
+}
+
+variable "kms_master_key_id" {
+  description = "ARN of the Customer Managed Key (CMK)."
+  type        = string
+  default     = ""
+}
+
+variable "sns_cmk_policy" {
```

Comment:
Another variable that I think is not even used.

Reply:
Perhaps I've been testing something. Thanks for noticing...